skimage/filters/tests/test_thresholding.py

```diff
@@ -262,6 +263,19 @@ def test_isodata_moon_image_negative_float():
                         [-13.83789062, -12.84179688, -11.84570312, 22.02148438,
                          23.01757812, 24.01367188, 38.95507812, 39.95117188])
 
+def test_threshold_minimum():
```

Comment:
We need to test "min", "mid" and "max" here, please. Ideally with some small test images where we can verify the behavior by hand.